### DISCUSSION TITLE
Xamarin Project with grial , dos it coz permission issue when internal sharing on Play store ?[Bug]

### DISCUSSION THREAD

**Discussion by ShamineDsouza** [xamarin/Xamarin.Forms#10907] [General]
Hi ,

I have implemented grial in my project and have almost completed development .
However , we need to do internal testing before proceeding . However I have been getting permission issue when trying the internal sharing uploading of google play store

Description
I am trying to load the .apk in the internal sharing link provided by google play store for organizational testing .

Basic Information
Xamarin

Screenshots
Hi ,

I have implemented grial in my project and have almost completed development .
However , we need to do internal testing before proceeding . However I have been getting permission issue when trying the internal sharing uploading of google play store

Description
I am trying to load the .apk in the internal sharing link provided by google play store for organizational testing .

Basic Information
Xamarin

Screenshots
![image](https://user-images.githubusercontent.com/25946929/83345561-3530ef80-a31d-11ea-81b9-ae1d5bf41fe0.png)

**Comment by jsuarezruiz** [comment]
You can see more information about private distribution here https://support.google.com/googleplay/android-developer/answer/9303479?hl=en

I would suggest checking a couple of things:
1. That you are an authorized uploader for internal app sharing. 
2. That you are using one of your published application IDs in the Google Play Console.
